### PR TITLE
clock: add background to moon

### PR DESCRIPTION
**Pull request by tylershuster**
Fixes https://github.com/urbit/landscape/issues/186

### CODE REVIEW

File: pkg/interface/src/views/apps/launch/components/tiles/clock.js [hunk 1]
```diff
@@ -202,9 +202,10 @@ class Clock extends React.PureComponent {
   }
 
   initGeolocation() {
-    if (typeof this.props.data === 'object') {
-
-      const { latitude: lat, longitude: lon } = this.props.data;
+    if (typeof this.props.data === 'string') {
+      const latlon = this.props.data.split(',');
+      const lat = latlon[0];
+      const lon = latlon[1];
```
Comment (matildepark) [inline pkg/interface/src/views/apps/launch/components/tiles/clock.js:208]: unrelated change?

Comment (tylershuster) [reply pkg/interface/src/views/apps/launch/components/tiles/clock.js:208]: looks like it don't it

**Review by matildepark — APPROVED** [review]
lgtm
